include/ginkgo/core/base/segmented_array.hpp

```diff
@@ -0,0 +1,245 @@
+// SPDX-FileCopyrightText: 2017 - 2024 The Ginkgo authors
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
```

Comment:
These update-every-file changes are always a bit annoying. I would pick up #1466 after the release, perhaps this could be addressed then to keep the disruption minimal.